### PR TITLE
Fix comments in setup script for QuickStart

### PR DESCRIPTION
**Pull request by tetafro**

### CODE REVIEW

File: deployments/quick-start/setup.sh [hunk 1]
```diff
@@ -46,14 +47,16 @@ then
...
 nohup kubectl port-forward $vaultPod 8200 &
 
-# Set environment variables and authenticate
+# 2.2 Set environment variables and authenticate
 export VAULT_ADDR=http://127.0.0.1:8200
 vault auth vault-root-token
 
-# Set up the Root Certificate Authority
+# 2.2 Set up the Root Certificate Authority
 # Create a Root CA that expires in 10 years:
```
Comment (F21) [inline deployments/quick-start/setup.sh]: This should be `2.3`. I got the numbering wrong when writing up `quick-start.md`.

**Review by F21 — CHANGES_REQUESTED** [review]
Can you also fix the rest of the numberings under the `Setup Vault` section? I'll push a change to `quick-start.md` to fix the numbering in that document. Once that's done, please squash the commit and we should be good to go! 🚢

**Comment by F21** [comment]
I've fixed the numbering in `quick-start.md` in #73 :smile:

**Comment by tetafro** [comment]
I've fixed the numbering in setup.sh.

**Review by F21 — APPROVED** [review]

**Comment by F21** [comment]
Perfect! Thanks for your contribution!

**Pull request merged**